templates/coderd.yaml

```diff
@@ -240,6 +240,10 @@ metadata:
   name: {{ include "coder.serviceName" . }}
   {{- end }}
   namespace: {{ .Release.Namespace | quote }}
+  annotations:
+  {{- range $key, $value := .Values.coderd.serviceAnnotations }}
+    {{ $key }}: {{ $value | quote }}
+  {{- end }}
```

Comment:
Is this OK if there's nothing in the map?

Reply:
Should be

Reply:
I'll try this